src/annotator/test/guest-test.js

```diff
@@ -44,22 +35,29 @@ const timeoutPromise = (millis = 0) =>
 
 describe('Guest', () => {
   const sandbox = sinon.createSandbox();
-  let CrossFrame;
-  let fakeCrossFrame;
   let highlighter;
   let guestConfig;
   let htmlAnchoring;
   let rangeUtil;
   let notifySelectionChanged;
 
+  let CrossFrame;
+  let fakeCrossFrame;
+
+  let DocumentMeta;
+  let fakeDocumentMeta;
+
+  let PdfIntegration;
+  let fakePdfIntegration;
```

Comment:
I've refactored the Guest tests to use a consistent approach to mocking these classes. The `CapitalCase` var is the constructor mock. The `fakeCamelCase` var is the mock instance.

Reply:
Thanks for the notes. I helps me learn about the conventions.